presentations/en/2019-11-05-Working-in-the-open.md

```diff
@@ -72,7 +72,6 @@ Note:
   - You trust your team
   - You trust this is good for the public
   - The public should be able to trust the government
-    - [Accountability](https://product-guide.18f.gov/Values/working-in-the-open/)
```

Comment:
They've basically changed their whole site. Do you think there's a way to trace where those values went? Is it gone gone or just moved somewhere else? It's unfortunate...

Reply:
I updated the link a few months back because it had moved.  But I looked on the Github and it's really gone.  Unless you can find it.  Google still finds the old link